torchaudio/models/_wavernn.py

```diff
@@ -50,56 +48,277 @@ def forward(self, x: Tensor) -> Tensor:
 
 class _MelResNet(nn.Module):
     r"""This is a MelResNet layer based on a stack of ResBlocks. It is a block used in WaveRNN.
-    WaveRNN is based on the paper "Efficient Neural Audio Synthesis". Nal Kalchbrenner, Erich Elsen,
-    Karen Simonyan, Seb Noury, Norman Casagrande, Edward Lockhart, Florian Stimberg, Aaron van den Oord,
-    Sander Dieleman, Koray Kavukcuoglu. arXiv:1802.08435, 2018.
 
     Args:
-        res_blocks: the number of ResBlock in stack (default=10).
-        input_dims: the number of input sequence (default=100).
-        hidden_dims: the number of compute dimensions (default=128).
-        output_dims: the number of output sequence (default=128).
-        pad: the number of kernal size (pad * 2 + 1) in the first Conv1d layer (default=2).
+        n_res_block: the number of ResBlock in stack (default=10)
+        n_freq: the number of bins in a spectrogram (default=128)
+        n_hidden: the number of hidden dimensions (default=128)
+        n_output: the number of output dimensions (default=128)
+        kernel_size: the number of kernel size in the first Conv1d layer (default=5)
 
     Examples::
-        >>> melresnet = _MelResNet(res_blocks=10, input_dims=100,
-                                   hidden_dims=128, output_dims=128, pad=2)
-        >>> input = torch.rand(10, 100, 512)
+        >>> melresnet = _MelResNet(n_res_block=10, n_freq=128, n_hidden=128,
+                                   n_output=128, kernel_size=5)
+        >>> input = torch.rand(10, 128, 512)
         >>> output = melresnet(input)
     """
 
-    def __init__(self, res_blocks: int = 10,
-                 input_dims: int = 100,
-                 hidden_dims: int = 128,
-                 output_dims: int = 128,
-                 pad: int = 2) -> None:
+    def __init__(self,
+                 n_res_block: int = 10,
+                 n_freq: int = 128,
+                 n_hidden: int = 128,
+                 n_output: int = 128,
+                 kernel_size: int = 5) -> None:
         super().__init__()
 
-        kernel_size = pad * 2 + 1
         ResBlocks = []
 
-        for i in range(res_blocks):
-            ResBlocks.append(_ResBlock(hidden_dims))
+        for i in range(n_res_block):
+            ResBlocks.append(_ResBlock(n_hidden))
```

Comment:
You can use one-liner here for better readability.

```
ResBlocks = [_ResBlock(n_hidden) for _ in range(n_res_block)]
```

Use underscore as variable name when you do not use the variable.

Reply:
Nice suggestion! fixed.